artifactory/commands/config.go

```diff
@@ -238,12 +234,14 @@ func (cc *ConfigCommand) getConfigurationFromUser() error {
 		}
 	}
 
-	cc.readRefreshableTokensFromConsole()
 	cc.readClientCertInfoFromConsole()
```

Comment:
I think that the client cetificates data should be at the very end of the interactive configuration process.